lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_version_helper.rb

```diff
@@ -89,15 +187,45 @@ def self.calc_next_release_base_version(version)
           { VERSION_NAME => "#{vp[MAJOR_NUMBER]}.#{vp[MINOR_NUMBER]}", VERSION_CODE => version[VERSION_CODE] }
         end
 
+        # Compute the name of the next version to use after code freeze, by incrementing the current version name and making it a `-rc-1`
+        #
+        # @example
+        #   calc_next_release_version({"name": "1.2", "code": 456}) #=> {"name":"1.3-rc-1", "code": 457}
+        #   calc_next_release_version({"name": "1.2.3", "code": 456}) #=> {"name":"1.3-rc-1", "code": 457}
+        #   calc_next_release_version({"name": "1.2", "code": 456}, {"name":"alpha-1.2", "code": 457}) #=> {"name":"1.3-rc-1", "code": 458}
+        #
+        # @param [Hash] version The current version hash, with keys `"name"` and `"code"`
+        # @param [Hash] alpha_version The current alpha version hash, with keys `"name"` and `"code"`, or nil if no alpha version
+        #
+        # @return [Hash] The hash containing the version name and code to use after release cut
+        #
         def self.calc_next_release_version(version, alpha_version = nil)
```

Comment:
This one confused me quite a bit… method named "next release version" but in fact it's returning a `"X.Y-rc-1"` version… so more the name of the "first beta after code freeze" rather than the "release version".

This is why the documentation was also so confusing to write at first. Will probably rename in a separate/future PR.